dist/functional-test.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+
+ETCD_IMG="quay.io/coreos/etcd:v3.0.3"
```

Comment:
Yes but on the other hand, you can start getting spurious failures since etcd is changing underneath you.